docs/src/index.md

```diff
@@ -99,33 +118,40 @@ The particle filter parameters are saved in the following data structure:
 ParticleDA.FilterParameters
 ```
 
```

Comment:
```suggestion
### Output statistics

There are four different output statistics options available:

* NaiveMeanSummaryStat
* NaiveMeanAndVarSummaryStat
* MeanSummaryStat
* MeanAndVarSummaryStat

Information on the differences in methods of calculation can be found in [statistics.jl](https://github.com/matt-graham/ParticleDA.jl/blob/mmg/flat-states/src/statistics.jl)

Please be advised that the MeanSummaryStat and MeanAndVarSummaryStat calculations relie on custom MPI reductions, which inhibits their use on non-Intel architectures. 
In this case please revert to using the Naive calulcations.
```

Reply:
I've [added some docstrings to the summary stat types](https://github.com/Team-RADDISH/ParticleDA.jl/blob/97f789acb9ab7fdae2534132b41636dce3d10b9a/src/statistics.jl) and added these docstrings plus a short explanation to [the _Usage_ section of the documents index page](https://github.com/Team-RADDISH/ParticleDA.jl/blob/97f789acb9ab7fdae2534132b41636dce3d10b9a/docs/src/index.md#usage).